doc/decimal/basics.adoc

```diff
@@ -23,7 +23,7 @@ constexpr decimal32(T coeff, T2 exp, bool sign = false) noexcept;
 Where types `T` and `T2` are integral types (signed and unsigned are both allowed).
 Lastly the sign follows the convention of `signbit` where `false` is positive and `true` is negative.
 If both a negative coefficient and a sign are passed then the resulting decimal number will be negative.
-The final number constructed is in the form coeff x 10^exp.
+The final number constructed is in the form (sign ? -1 : 1) x coeff x 10^exp.
```

Comment:
Actually, this is still incorrect, as this would render a positive decimal number when both `sign == true` and `coeff < 0`.

```c++
(sign && coeff >= 0 ? -1 : 1) x coeff x 10^exp
```

I think this also takes care for the negative zero case.

Reply:
How about: 

```
(sign || coeff < 0 ? -1 : 1) x coeff x 10^exp
```

Which matches the actual constructor logic:
```
if (coeff < 0 || sign)
      {
          bits_ |= detail::d32_sign_mask;
          isneg = true;
      }
```

With (sign && coeff >= 0) the assignment (true, -1) would yield 1 when it should be -1.